Add id for the bar container

babouinette opened the pull request:
Thank you for this plugin. Really simple to use :)

I think adding an Id to the bar container would allow easy customization with CSS (for example to change the position of the bar => above bottom navbar on mobile, or to hide it easily by adding a hidden class if the mobile menu is open)

What do you think about it?

thigoap commented:
Hi, thank you for the contribution!

I didn't think about using css as a solution for customization to avoid the user having to write even more code, in addition to the shortcode and the &lt;div class="reader-bar-start"&gt;.
So I added the basic customization of thickness and colors directly in the plugin via the shortcode.

But it is a good idea, if someone wants more flexibility in customization, as you exemplified, to change the position of the bar, which is not possible only with the shortcode.

I'll do some tests to check if, if customization is done by the plugin shortcode **and also** by the css, which one of the two will prevail.

babouinette commented:
Hello,

Thank you for taking a look.

I tried it with a shortcode : 

`{% readerBar "25px" %}
`

which creates : 
```
<div style="height:25px;width:100%;background-color:black;position:fixed;top:2px;left:0;z-index:100;transition:0.2s;">
    <div id="readerBar" style="height: 25px; width: 100%; background-color: orange; position: fixed; top: 2px; left: 0px; z-index: 200; transition: all 0.2s ease 0s;"></div>
  </div>
```

By adding in my css :
```
#readerBar{
    background-color: blue !important;
    height: 10px !important;
}
```
it restyles the div (height and color) with id "readerBar" (it needs the **!important** in the css to override the inline style).

It should work the same with adding a readerBarContainer id to the outer div.

thigoap commented:
Indeed, inline CSS prevails over external CSS, but **!important** solves this problem.

I will merge your PR now, and, probably untill the end of this month I will publish a new v0.2.0 of the plugin with a new little feature, and also with the id in the outher div.

Thanks again for the contribution, and happy coding!
